maintainers/maintainer-list.nix

```diff
@@ -13426,4 +13426,10 @@
     github = "jpagex";
     githubId = 635768;
   };
+  vbrandl = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Done in df432c38cfb